test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildALibrary.cs

```diff
@@ -556,6 +557,37 @@ public void It_fails_gracefully_if_targetframework_is_empty()
                 .And.NotHaveStdOutContaining(">="); // old error about comparing empty string to version
         }
 
+        [Theory]
+        [InlineData("netcoreapp2.1")]
+        [InlineData("netstandard2.1")]
```

Comment:
Hardcoding these versions to 2.1 in this test means we'll have to update the test data each time we start supporting a new framework.  Alternatively, we could target e.g. `netcoreapp100.0` in these tests, but then the test is not as close to the real scenario.